app/src/main/java/org/sopt/official/App.kt

```diff
@@ -65,6 +66,6 @@ class App : Application() {
     }
 
     private fun initFlipper() {
-        FlipperInitializer.init(this)
+        FlipperInitializer
```

Comment:
```suggestion
        FlipperInitializer.init(this)
```

기존 코드 그대로 가져가야되지 않을까요?

Reply:
넵! 수정하겠습니다.